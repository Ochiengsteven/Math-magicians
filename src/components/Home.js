import React from 'react';

const Home = () => (
  <div>
    <h3>Welcome to Math Magicians</h3>
    <p>
      Explore the fascinating world of mathematics! Whether you are a student, an educator,
      or simply curious about math, Math Magicians has something for everyone. From
      basic calculations to advanced concepts, we have got you covered.
    </p>
  </div>
);

export default Home;
