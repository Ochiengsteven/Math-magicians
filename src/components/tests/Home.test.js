import React from 'react';
import { render } from '@testing-library/react';
import Home from '../Home';

describe('Home component', () => {
  it('should match the snapshot', () => {
    const { asFragment } = render(<Home />);
    expect(asFragment()).toMatchSnapshot();
  });
});
