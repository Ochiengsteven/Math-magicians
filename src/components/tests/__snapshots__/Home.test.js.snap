// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Home component should match the snapshot 1`] = `
<DocumentFragment>
  <div
    class="content"
  >
    <h3>
      Welcome to Math Magicians
    </h3>
    <p>
      Explore the fascinating world of mathematics! Whether you are a student, an educator, or simply curious about math, Math Magicians has something for everyone. From basic calculations to advanced concepts, we have got you covered.
    </p>
    <p>
      Our mission is to make math engaging and accessible to all. We offer a variety of resources, including a powerful calculator that can handle complex equations, as well as inspiring quotes to motivate your mathematical journey.
    </p>
    <p>
      Dive into our calculator to perform arithmetic operations, solve equations, and more. Test your mathematical skills and challenge yourself with our user-friendly interface. Whether you're a math enthusiast or just need some quick calculations, Math Magicians is here to assist you.
    </p>
    <p>
      And don't forget to check out our collection of thought-provoking math quotes. Find inspiration from renowned mathematicians and thinkers. Let their words ignite your passion for learning and exploring the endless possibilities of mathematics.
    </p>
  </div>
</DocumentFragment>
`;
